projects/spring-framework/jdbc/MapperFuzzerServer.java

```diff
@@ -0,0 +1,27 @@
+package jdbc;
```

Comment:
Missing Copyright here and following .java files

Reply:
Added in https://github.com/google/oss-fuzz/pull/8359/commits/3fee0cfc8fd8ee7f05701fd86ae1a717d7324c03